website/src/components/audio-player.tsx

```diff
@@ -3,6 +3,25 @@ import { FiLoader } from "react-icons/fi/index"
 import { MdPauseCircleOutline, MdPlayCircleOutline } from "react-icons/md/index"
 import * as css from "./audio-player.css"
 
+/**
+ * Get our default load status for the audio player.
+ * On some user agents, we need to default to "loaded" because audio can never be loaded in the background.
+ */
+function getDefaultLoadStatus() {
+  // TODO: is there a place to store this instead of doing this check everytime?
+  // happens at this level because vite will run this server-side and crash sometimes
```

Comment:
My first thoughts are to do this through React state. I figure you considered that already though. Is there anything worse about using state than running this every time?

Reply:
I'll look for a way to see if we are on the server-side, and short-circuit if we are? I wonder if that would let us make it a constant.